configs/NVIDIA/nemo-release.yaml

```diff
@@ -0,0 +1,26 @@
+target_repository:
+  HTTPS: https://github.com/NVIDIA/NeMo.git
+  # checkout a particular branch or a tag
+  checkout: main
+  # define installing package extras
+  install_extras: all
+  # copy some  test from the target repository
+  copy_tests:
+    - tests
+
+before_install:
+  - sudo apt-get install libsndfile1
+
+dependencies:
+  - name: pytorch-lightning
+    HTTPS: https://github.com/PyTorchLightning/pytorch-lightning.git
+    checkout: release/1.5.x
+    install_extras: loggers
+
+testing:
+  dirs:
+    - tests
+  # additional pytest arguments
+  pytest_args: -m "not pleasefixme and not torch_tts" --cpu --with_downloads --relax_numba_compat
```

Comment:
@SeanNaren why is this? :D

Reply:
Maybe @ericharper could explain the additional parameters here, was forwarded it from the slack channel!

Reply:
I am wondering what the commands shall do `--cpu --with_downloads --relax_numba_compat`
when I call `pytest --help`  none of them is there... do you need dome special extension?

Reply:
@Borda I think you can define such things in a conftest.py file like here: https://github.com/NVIDIA/NeMo/blob/main/tests/conftest.py#L30